README.md

```diff
@@ -1,33 +1,35 @@
 # mongoose-harakiri
 
-**Mongoose Harakiri ensures a stable connection with mongoose, and kills itself if it can't establish one.**
+**Mongoose Harakiri ensures a stable connection with mongoose, and kills itself if it can't establish one or loses it.**
```

Comment:
"or loses it" sounds like it's a default behavior, but the default is reconnection when it loses it. so maybe we can reword this to emphasize that it's optional?

Reply:
Oh, and I would say that not a big chunk of users use clusters, and this is a bug anyway, so this option should be false by default in that over time when the library fixes reconnection logic to clusters, no one needs to make a change.